app/src/main/java/com/infomaniak/mail/ui/MainViewModel.kt

```diff
@@ -415,7 +415,7 @@ class MainViewModel @Inject constructor(
             updateFolders(mailbox)
 
             // Refresh Threads
-            (currentFolderId?.let(folderController::getFolder) ?: folderController.getFolder(DEFAULT_SELECTED_FOLDER))
+            (currentFolderId?.let { folderController.getFolder(it) } ?: folderController.getFolderBlocking(DEFAULT_SELECTED_FOLDER))
```

Comment:
Revert the `let` part, we prefer the previous way of writing it.

Reply:
Doesn't compile either